src/metatensor/models/utils/logging.py

```diff
@@ -54,72 +54,77 @@ def __init__(
         else:
             self.only_one_energy = False
 
-        # Save the model capabilities for later use:
+        # Save the model capabilities. This will be useful to know
+        # what physical quantities we are printing
         self.model_capabilities = model_capabilities
 
     def log(
         self,
-        epoch: int,
-        train_loss: float,
-        validation_loss: float,
-        train_info: Dict[str, float],
-        validation_info: Dict[str, float],
+        metrics: Union[Dict[str, float], List[Dict[str, float]]],
+        epoch: Optional[int] = None,
     ):
         """
-        Log the training metrics.
+        Log the metrics.
 
-        The training metrics are automatically aligned to make them easier to read,
-        based on the order of magnitude of each metric at the start of the training.
+        The metrics are automatically aligned to make them easier to read,
+        based on the order of magnitude of each metric given to the class
+        at initialization.
 
-        :param epoch: The current epoch.
-        :param train_loss: The current training loss.
-        :param validation_loss: The current validation loss.
-        :param train_info: The current training metrics.
-        :param validation_info: The current validation metrics.
+        :param metrics: The current metrics to be printed.
+        :param epoch: The current epoch (optional).
```

Comment:
What happens if this is `None`? Please add this to the docstring.

Reply:
Done